src/panels/config/integrations/integration-panels/ozw/dialog-ozw-refresh-node.ts

```diff
@@ -0,0 +1,292 @@
+            `
+          : html`
+              ${this._active
+                ? html`
+                    <div class="flex-container">
+                      <ha-circular-progress active></ha-circular-progress>
+                      <div>
+                        <p>
+                          <b>
+                            ${this.hass.localize(
+                              "ui.panel.config.ozw.refresh_node.refreshing_description"
+                            )}
+                          </b>
+                        </p>
+                        <p>
+                          ${this.hass.localize(
+                            "ui.panel.config.ozw.refresh_node.node_status"
+                          )}:
+                          ${this._node!.node_query_stage}
+                          (${this.hass.localize(
+                            "ui.panel.config.ozw.refresh_node.step"
+                          )}
+                          ${this._progress.indexOf(
+                            this._node!.node_query_stage
+                          ) + 1}/17)
+                        </p>
+                        <p>
+                          <em>
+                            ${this.hass.localize(
+                              "ui.panel.config.ozw.node_query_stages." +
+                                this._node!.node_query_stage.toLowerCase()
+                            )}</em
+                          >
+                        </p>
+                      </div>
+                    </div>
+                  `
+                : html`
+                    ${this.hass.localize(
+                      "ui.panel.config.ozw.refresh_node.description"
+                    )}
+                    <p>
+                      ${this.hass.localize(
+                        "ui.panel.config.ozw.refresh_node.battery_note"
+                      )}
+                    </p>
+                  `}
+              ${this._nodeMetaData?.WakeupHelp !== ""
+                ? html`
+                    <b>
+                      ${this.hass.localize(
+                        "ui.panel.config.ozw.refresh_node.wakeup_header"
+                      )}
+                      ${this._nodeMetaData?.Name}
+                    </b>
+                    <blockquote>
+                      ${this._nodeMetaData?.WakeupHelp}
+                      <br />
+                      <em>
+                        ${this.hass.localize(
+                          "ui.panel.config.ozw.refresh_node.wakeup_instructions_source"
+                        )}
+                      </em>
+                    </blockquote>
+                  `
+                : ""}
+              ${!this._active
+                ? html`
+                    <mwc-button @click=${this._startRefresh}>
+                      ${this.hass.localize(
+                        "ui.panel.config.ozw.refresh_node.start_refresh_button"
+                      )}
+                    </mwc-button>
```

Comment:
Put this in the actions of the dialog, align it right and don't color it red

Reply:
The red from the screenshots is just the primary theme color for my dev instance :)